addons/mail/static/src/less/composer.less

```diff
@@ -8,7 +8,80 @@
         margin-right: 30px;
     }
 }
+.o_modal_fullscreen {
+    padding: 0;
+    .o_viewer_content {
```

Comment:
Structure is strange, I have a `.o_viewer_content` inside a `.o_viewer_content`.